src/main/kotlin/org/jitsi/jicofo/jibri/BaseJibri.kt

```diff
@@ -72,6 +72,7 @@ abstract class BaseJibri internal constructor(
 
     fun handleJibriRequest(request: JibriRequest): IqProcessingResult =
         if (accept(request.iq)) {
+            logger.info("Accepted jibri request: ${request.iq.toXML()}")
```

Comment:
Most "ordinary" XMPP logging is done at either `debug` or `trace` level - do you really want this as `info`?

Reply:
Yes, I erred on the side of logging too much here, because it's hard to log all the useful information otherwise (is the request coming from a client or a jibri, which client/jibri is it coming from, is it a request or status update, is there an error condition, etc). Jibri messages tend to be small and not that many, so the volume should be okay.